CHANGELOG.rst

```diff
@@ -16,13 +16,18 @@ Subsections for each version can be one of the following;
 
 Each individual change should have a link to the pull request after the description of the change.
 
+1.2.1 (2024-02-08)
+------------------
+Added
+^^^^^
+- Updated GroupRareLevelsTransformer so that when working with category dtypes it forgets categories encoded as rare `#177 https://github.com/lvgig/tubular/pull/177`_
```

Comment:
"...it forgets categories which have been encoded as rare and so are no longer present in the data"

^without the last bit it wasn't clear to me on first read.

Reply:
thanks, updated here! https://github.com/lvgig/tubular/pull/177/commits/227519e8d55482856b16ef437a7077e3d452037f